src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -77,7 +78,7 @@ module CanopyFluxesMod
   logical,  public :: perchroot_alt = .false.  
   !
   ! !PRIVATE DATA MEMBERS:
-  logical, private :: use_undercanopy_stability = .true.      ! use undercanopy stability term or not
```

Comment:
Since by default this is set in the namelist, the default in the code doesn't really matter here.

Reply:
Nothing needs to be done here.